### ISSUE TITLE
Use correct script tags

### ISSUE THREAD

**Issue by stevepolitodesign** [thoughtbot/suspenders#1111]
## Steps to reproduce

1. Checkout the [support_rails_7](https://github.com/thoughtbot/suspenders/tree/support_rails_7) branch.
2. Once you are in the `suspenders` directory, run `./bin/suspenders some_app_name` to generate a Rails app from the current state of suspenders.
3. Change directories into `some_app_name`.
4. Run `rails g scaffold some_model some_attribute` followed by `rails db:migrate`.
5. Run `rails s` and visit a route for the resource created in step 4.

<img width="1010" alt="CleanShot 2022-12-14 at 10 00 19@2x" src="https://user-images.githubusercontent.com/5122678/207671241-618445a1-a9c9-49d3-b85a-67cb501aaaa8.png">

**Comment by stevepolitodesign** [comment]
This is what ships with a new Rails 7 app generated with `-c=postcss`

```html+erb
  <head>
    <title>Rails7Css</title>
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <%= csrf_meta_tags %>
    <%= csp_meta_tag %>

    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
  </head>
```

**Comment by stevepolitodesign** [comment]
Closed by #1113
